pkg/cloudprovider/provider/equinixmetal/types/types.go

```diff
@@ -26,6 +26,7 @@ type RawConfig struct {
 	ProjectID    providerconfigtypes.ConfigVarString   `json:"projectID,omitempty"`
 	BillingCycle providerconfigtypes.ConfigVarString   `json:"billingCycle"`
 	InstanceType providerconfigtypes.ConfigVarString   `json:"instanceType"`
+	Metro        providerconfigtypes.ConfigVarString   `json:"metro"`
```

Comment:
Shouldn't this be an optional field? Basically, both Facilities and metro should be made optional and then we can check in `Validate` that at least one of them is specified. WDYT?

Reply:
I do this already https://github.com/kubermatic/machine-controller/pull/1339/files#diff-86dce1d2d7e8fb9dfddab19175ce5b62bb771587033f0bb6a3b83961b9318e75R195

Reply:
Oh actually that's true, will adjust that now